### DISCUSSION TITLE
How to improve imputation accuracy ?

### DISCUSSION THREAD

**Discussion by sidevshiy** [maize-genetics/phg_v2#223] [General]
Hello,

Could you please advise on which parameters in the process of building and imputing play a major role in improving imputation accuracy? For example, would splitting the reference file into more ranges (since I have long intergenic regions) help, or should I focus on certain parameters during the map-kmers or path-finding steps? Which of these should I prioritize for better results?

**Comment by zrm22** [comment]
Hello,

We are actually currently investigating a bug in the imputation code which is leading some users to have very poor imputation accuracies.  Some of the parameters may be dropped or adjusted but right now the biggest parameter which might be having an effect is map-kmer's --min-proportion-same-reference-range.

We have run some pilot studies with older versions of PHGs and splitting the reference ranges up did not seem to improve the accuracies.  You are welcome to try it to see, but it would require a rebuild(you could skip the align-assemblies step however).

We are hoping that we can resolve this bug quickly and will put out a new release when it is done.
